meeting-notes/2016-09-26-libp2p.md

```diff
@@ -0,0 +1,58 @@
+## libp2p
+
+#### Lead: @dignifiedquire
+#### Notetaker: @lgierth
+
+#### Participants
+
+- @dignifiedquire
+- @lgierth
+- @kubuxu
+- @flyingzumvalt
+- @em-ly
+- @keks
+- @gavinmcdermott
+
+## Agenda
+
+Waffle Board: https://waffle.io/libp2p/libp2p
+Roadmap: https://github.com/libp2p/libp2p/blob/master/ROADMAP.md
+https://project-repos.ipfs.io/ source: https://github.com/ipfs/project-repos
+
+
+## Notes
+
+- from david
+  - Hi (David), same goes for this meeting, I can't be in the zoom call, so here is a bunch of notes too :D 
+  - I've also processed the libp2p waffle board and with regards to:
+  - ms1: just a few low hanging fruit issues. 
+  - ms2: the one that needs to be 'revisited', don't mind its existence
+  - ms3: seems that we are just missing a couple of things, right @richardlitt? :D :
+    - two standardized readmes
+    - create https://github.com/libp2p/community and literally just point to ipfs/community 
+    - make project-repos.ipfs.io to green across the board for the libp2p repos
+    - seems that we are two readmes down from completing MS3, right @richardlitt? :D 
```

Comment:
I think those are done? I need review from other people, and then this is closed.